src/shared/lib/StoreUtils.ts

```diff
@@ -120,6 +120,18 @@ export const MolecularAlterationType_filenameSuffix: {
     STRUCTURAL_VARIANT: 'structural_variants',
 };
 
+export async function chunkCalls<ParamData, ResponseData>(
```

Comment:
we need to move this out to utils package because it may need to be used by other packages that fetch their own data